File: JINSEOP/프로그래머스_WEEK15/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = false;
    
    if(s.size() == 4 | s.size() == 6){
        for(char c : s){
            if(c < '0' || c > '9'){
                answer = false;
                break;
            }
            answer = true;
        }
    }
    
    return answer;
}

File: DONGUK/프로그래머스_WEEK_15/문자열 다루기 기본.py
def solution(s):
    answer = True
    
    if len(s) != 4 and len(s) != 6:
        return False
    
    for i in range(len(s)):
        if '0' <= s[i] <= '9':
            continue
            
        else:
            return False
        
    return answer